addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/UrlBruteForce.java

```diff
@@ -297,39 +302,39 @@ public String requester(HttpMessage msg, Method method, Map<String, String> para
     public ParamReasons compare(
             ComparableResponse resp1, ComparableResponse resp2, Map<String, String> params) {
         if (resp1.getStatusCode() != resp2.getStatusCode()
-                || ComparableResponse.statusCodeHeuristic(resp1, resp2) < 1) {
-            return new ParamReasons("http code", params);
+                || ComparableResponse.statusCodeHeuristic(resp1, resp2) < 0.6) {
+            return new ParamReasons(Reason.HTTP_CODE, params);
         }
 
         if (ComparableResponse.headersCompareHeuristic(resp1, resp2) < 0.6) {
-            return new ParamReasons("http headers", params);
+            return new ParamReasons(Reason.HTTP_HEADERS, params);
         }
 
         if (!Objects.equals(
                 resp1.getHeaders().get("Location"), resp2.getHeaders().get("Location"))) {
-            return new ParamReasons("redirect", params);
+            return new ParamReasons(Reason.REDIRECT, params);
         }
 
         if (ComparableResponse.bodyTreesStructureHeuristic(resp1, resp2) < 0.6) {
-            return new ParamReasons("body heuristic mismatch", params);
+            return new ParamReasons(Reason.BODY_HEURISTIC_MISMATCH, params);
         }
 
         if (ComparableResponse.lineCountHeuristic(resp1, resp2) < 0.6) {
-            return new ParamReasons("line count mismatch", params);
+            return new ParamReasons(Reason.LINE_COUNT, params);
         }
 
         if (ComparableResponse.wordCountHeuristic(resp1, resp2) < 0.6) {
-            return new ParamReasons("word count mismatch", params);
+            return new ParamReasons(Reason.WORD_COUNT, params);
         }
 
-        if (resp1.compareWith(resp2) < 1) {
-            return new ParamReasons("body mismatch", params);
-        }
+        // if (resp1.compareWith(resp2) < 1) {
+        //     return new ParamReasons("body mismatch", params);
+        // }
```

Comment:
Is this block in or out? (Or is that still up for testing/future PR)

Reply:
Future PR.

Reply:
👍